### PR TITLE
NDS-886: Replace etcddumper in backup.sh with something that actually works

### PR DESCRIPTION
**Pull request by bodom0015**
See https://opensource.ncsa.illinois.edu/jira/browse/NDS-886

### CODE REVIEW

File: scripts/backup.sh [hunk 1]
```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
 [ $DEBUG ] && set -x
 
+if [ "${CLUSTER_ID}" == "" ];
+	echo "You must specify a CLUSTER_ID"
```
Comment (craig-willis) [inline scripts/backup.sh]: Not sure why this worked for you, but I needed an explicit "then"

File: scripts/retrieve-backup.sh [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+
 # XXX: Set this to "echo" to for a dry-run
 DEBUG=""
 
 # Grab date / cluster name
-IFS='-' read -ra HOST <<< "${HOSTNAME:-localhost}"
-TARGET_PATH=${BACKUP_DEST:-/ndsbackup}/${HOST[0]}/$1
+TARGET_PATH=${BACKUP_DEST:-/ndsbackup}/${CLUSTER_ID}/$1
 
 echo "Retrieving backup $1 for ${HOST[0]}: ${DATE}"
```
Comment (craig-willis) [inline scripts/retrieve-backup.sh]: Should this be CLUSTER_ID instead of HOST[0]?

File: scripts/list-backups.sh [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
 # XXX: Set this to "echo" to for a dry-run
 DEBUG=""
 
 # Grab date / cluster name
 DATE=$(date +%y-%m-%d.%H%M)
-IFS='-' read -ra HOST <<< "${HOSTNAME:-localhost}"
-TARGET_PATH=${BACKUP_DEST:-/ndsbackup}/${HOST[0]}
+TARGET_PATH=${BACKUP_DEST:-/ndsbackup}/${CLUSTER_ID}
 
 echo "Listing known backups for ${HOST[0]}:"
```
Comment (craig-willis) [inline scripts/list-backups.sh]: Should this be CLUSTER_ID instead of HOST[0]?

File: scripts/retrieve-backup.sh (hunk 2 above)
Comment (bodom0015) [reply scripts/retrieve-backup.sh]: I forget, but you are likely correct.

My attention to detail has been slipping lately :/

**Pull request merged**